tests/rptest/tests/cloud_storage_inventory_test.py

```diff
@@ -80,17 +82,20 @@ def test_load_inventory_report(self):
         self.start_redpanda()
         self._create_initial_topics()
         self.populate_bucket()
-        self._assert_hashes_are_found_on_leaders()
+        wait_until(lambda: self._hashes_are_found_on_leaders(),
+                   timeout_sec=60,
+                   backoff_sec=5)
 
-    def _assert_hashes_are_found_on_leaders(self):
+    def _hashes_are_found_on_leaders(self):
         found = set()
         expected = set()
         for spec in self.topics:
             for pid in range(spec.partition_count):
                 expected.add(NTP(ns="kafka", topic=spec.name, partition=pid))
         for node in self.redpanda.started_nodes():
             found |= self._find_ntp_hashes_on_node(node)
-        assert found == expected, f"{found=} |= {expected=}"
+        self.logger.debug(f"{found=} != {expected=}")
```

Comment:
nit: this debug log should be `self.logger.debug(f"{found=}, {expected=}")`, since `found == expected` once the hashes are indeed found.

Reply:
moved the log around without looking too closely. fixed, thanks